src/pages/NewChatPage.tsx

```diff
@@ -69,25 +69,24 @@ function useOptions({isGroupChat}: NewChatPageProps) {
             [],
             true,
         );
-        const maxParticipantsReached = selectedOptions.length === CONST.REPORT.MAXIMUM_PARTICIPANTS;
 
         const headerMessage = OptionsListUtils.getHeaderMessage(
             filteredOptions.personalDetails.length + filteredOptions.recentReports.length !== 0,
             Boolean(filteredOptions.userToInvite),
             debouncedSearchTerm.trim(),
-            maxParticipantsReached,
+            false,
```

Comment:
Let's get rid of this? The default value is `false`.

Reply:
Missed that, thanks! But maybe the whole param can go? Let's talk in the other thread about that...